config/environments/development.rb

```diff
@@ -1,33 +1,44 @@
-Lims3::Application.configure do
-  # Settings specified here will take precedence over those in config/application.rb
+Rails.application.configure do
+  # Settings specified here will take precedence over those in config/application.rb.
 
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
-  # Show full error reports and disable caching
+  # Do not eager load code on boot.
+  config.eager_load = false
+
+  # Show full error reports and disable caching.
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
-  # Don't care if the mailer can't send
+  # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = true
```

Comment:
Sure is.  Turns out it's been duplicated since c55e3a8 in January 2013.  The earlier one is where it's found in a fresh rails app, while the latter one is with the other email configuration added for this app.